plaid/src/apis/github/copilot.rs

```diff
@@ -0,0 +1,104 @@
+
+        let address = format!("/org/{organization}/copilot/billing/seats");
+
+        match self.make_generic_get_request(address, &module).await {
+            Ok((status, Ok(body))) => {
+                if status == 200 {
+                    Ok(body)
+                } else {
+                    Err(ApiError::GitHubError(GitHubError::UnexpectedStatusCode(
+                        status,
+                    )))
+                }
+            }
+            Ok((_, Err(e))) => Err(e),
+            Err(e) => Err(e),
+        }
+    }
+
+    // Add users to the Copilot subscription for an organization
+    // See https://docs.github.com/en/rest/copilot/copilot-user-management?apiVersion=2022-11-28#add-users-to-the-copilot-subscription-for-an-organization for more details
+    pub async fn add_users_to_org_copilot(&self, params: &str, module: &str) -> Result<u32, ApiError> {
+        #[derive(Deserialize, Serialize)]
+        struct Request {
+            org: String,
+            selected_usernames: Vec<String>,
+        }
+        let request: Request =
+            serde_json::from_str(params).map_err(|_| ApiError::BadRequest)?;
+
+        let organization = self.validate_org(&request.org)?;
+        for username in request.selected_usernames.iter() {
+            self.validate_username(&username)?;
+        }
+
+        info!("Adding users {:?} to Copilot subscription for org {organization}", request.selected_usernames);
+
+        let address = format!("/org/{organization}/copilot/billing/selected_users");
+
+        match self.make_generic_post_request(address, &request, &module).await {
+            Ok((status, Ok(_))) => {
+                if status == 201 {
+                    Ok(0)
+                } else {
+                    Err(ApiError::GitHubError(GitHubError::UnexpectedStatusCode(
+                        status,
+                    )))
+                }
+            }
+            Ok((_, Err(e))) => Err(e),
+            Err(e) => Err(e),
+        }
+    }
+
+    // Remove users from the Copilot subscription for an organization
+    // See https://docs.github.com/en/rest/copilot/copilot-user-management?apiVersion=2022-11-28#remove-users-from-the-copilot-subscription-for-an-organization for more details
+    pub async fn remove_users_from_org_copilot(&self, params: &str, module: &str) -> Result<u32, ApiError> {
+        #[derive(Deserialize, Serialize)]
+        struct Request {
+            org: String,
+            selected_usernames: Vec<String>,
+        }
+        let request: Request =
+            serde_json::from_str(params).map_err(|_| ApiError::BadRequest)?;
+
+        let organization = self.validate_org(&request.org)?;
+        for username in request.selected_usernames.iter() {
+            self.validate_username(&username)?;
+        }
+
+        info!("Remove users {:?} from Copilot subscription for org {organization}", request.selected_usernames.clone());
```

Comment:
You should be able to borrow here instead of cloning

Reply:
Fixed